### PR TITLE
vaex-ml added

### PR DESCRIPTION
**Pull request by maartenbreddels**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/vaex-ml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - vaex.ml
+
+about:
+  home: https://www.github.com/vaexio/vaex
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: Machine learning support for vaex
+  description: |
+    UNKNOWN
```
Comment (mariusvniekerk) [inline recipes/vaex-ml/meta.yaml]: Probably needs some kind of description here

File: recipes/vaex-ml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{% set name = "vaex-ml" %}
+{% set version = "0.5.0" %}
+{% set sha256 = "ae1f2c50c4883e42c99922351f2903ff7ad4078085c606ce7411b0696f88c1f2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
```
Comment (mariusvniekerk) [inline recipes/vaex-ml/meta.yaml]: you dont need the fn line anymore

**Comment by maartenbreddels** [comment]
Thanks, Marius, hope it's better now!

File: recipes/vaex-ml/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python
```
Comment (xhochy) [inline recipes/vaex-ml/meta.yaml]: Do you really need a build section? If so, better add a comment.

Comment (maartenbreddels) [reply recipes/vaex-ml/meta.yaml]: No, i guess no, thanks!

**Comment by maartenbreddels** [comment]
Thanks @xhochy and @mariusvniekerk!

**Pull request merged**